.github/workflows/ci.yml

```diff
@@ -34,5 +35,48 @@ jobs:
               run: cargo clippy --all-targets --all-features -- -D warnings
             - if: matrix.conf == 'check'
               run: cargo check --all-targets --all-features
-            - if: matrix.conf == 'test'
-              run: cargo test --all-targets --all-features
+
+    ci-test:
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                rust: [stable, beta]
+                experimental: [true, false]
+                exclude:
+                    - rust: stable
+                      experimental: true
+                    - rust: beta
+                      experimental: false
+        continue-on-error: ${{ matrix.experimental }}
+
+        services:
+            postgres:
+                image: postgres
```

Comment:
I guess this will use `postgres:latest`, right? This will then be a Postgres version 13. In the current development/production environment the version is 11.8. Shall we test here against the actual used version?

Reply:
We definitely _can_ do that.

Two questions regarding that:
1. Why are we not using version 13 in RDS?
2. Do we care about the exact version?

Reply:
1. The Postgres version 13 is not available in production RDS (latest is 12.4-R1).
2. Depends if we use the latest features, but I think it would make sense to do testing for our actual target platform.

Reply:
2. okay :)
1. Can we update to 12? Or is this difficult?

Reply:
3. The update is possible, but needs some manual work. I would prefer to only upgrade if we need a latest feature.

Reply:
Okay